src/components/members/MemberScreen.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { useRouteMatch } from 'react-router';
+import AccountCircleIcon from '@material-ui/icons/AccountCircle';
+import { Typography, Box } from '@material-ui/core';
+import { List } from 'immutable';
+import { buildMemberPath } from '../../config/paths';
+import membersData from '../../data/membersData';
+import { formatDate } from '../../utils/date';
+import { getMembershipsByMemberId } from '../../utils/member';
+import { insertCreatorWithItems } from '../../utils/item';
+import itemData from '../../data/itemData';
+import ItemsTable from '../items/ItemsTable';
+
+const MemberScreen = () => {
+  const match = useRouteMatch(buildMemberPath());
+  const memberId = match?.params?.memberId;
+  const member = membersData.find(({ id }) => id === memberId);
+  const itemsWithCreators = insertCreatorWithItems(itemData);
+
+  const memberships = getMembershipsByMemberId(member?.id);
```

Comment:
Can you use `memberId`?

Reply:
Everything related to the mock data has been removed in #12